browser/permissions/chrome_permission_origin_lifetime_monitor.cc

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/permissions/chrome_permission_origin_lifetime_monitor.h"
+
+#include "base/stl_util.h"
+#include "brave/browser/ephemeral_storage/ephemeral_storage_tab_helper.h"
+#include "chrome/browser/profiles/profile.h"
+#include "content/public/browser/render_frame_host.h"
+#include "content/public/browser/tld_ephemeral_lifetime.h"
+#include "content/public/browser/web_contents.h"
+#include "net/base/features.h"
+#include "net/base/url_util.h"
+
+namespace permissions {
+
+ChromePermissionOriginLifetimeMonitor::ChromePermissionOriginLifetimeMonitor(
```

Comment:
why is this named Chrome* instead of Brave*? That's very confusing to me because we generally use Brave* to distinguish between our implementations and upstream